### PR TITLE
VideoPress: Add inspector controls

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Adds the remaining controls to the sidebar - except Poster and Video File Settings
* Only show sidebar controls once the video is present in the block

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?

NA

#### Testing instructions:

* Add a new VP
* Add a video
* Check the new settings on the sidebar only appears after the video is added to the block

Captura de tela de 2022-06-28 15-21-37

Make sure they all work properly

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202518480088382

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/videopress/v6/components/inspector-controls.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+				<span>{ label }</span>
+			</Tooltip>
+		);
+	};
+
+	const handleAttributeChange = attributeName => {
+		return newValue => {
+			setAttributes( { [ attributeName ]: newValue } );
+		};
+	};
```
Comment (retrofox) [inline projects/plugins/jetpack/extensions/blocks/videopress/v6/components/inspector-controls.js]: Probably we'd like to memoize this function via using useCallback() hook:

```es6
const handleAttributeChange = useCallback(
	attributeName => {
		return newValue => {
			setAttributes( { [ attributeName ]: newValue } );
		};
	},
	[ setAttributes ]
);
```

**Review by retrofox — DISMISSED** [review]
LGTM.
I left a comment about wrapping the handler by using the useCallback() hok, but its up to you. We can iterate in follow up tasks.

File: projects/plugins/jetpack/extensions/blocks/videopress/v6/components/inspector-controls.js (hunk 1 above)
Comment (leogermani) [reply projects/plugins/jetpack/extensions/blocks/videopress/v6/components/inspector-controls.js]: Implemented your suggestion and tested it. can you approve again?

**Review by retrofox — APPROVED** [review]

**Comment by leogermani** [comment]
r248126-wpcom

**Pull request merged**